README.md

```diff
@@ -106,16 +118,27 @@ git submodule update --init --recursive
 ```
 
 ## Development
+
 See [CONTRIBUTING.md](CONTRIBUTING.md).
 
-## Third party tools
+## Third party tools and integrations
 
-There are also community tools that use OSV. Note that these are community built tools and
-unsupported by the core OSV maintainers.
+There are also community tools that use OSV. Note that these are community built
+tools and unsupported by the core OSV maintainers.
 
-- [G-Rath/osv-detector](https://github.com/G-Rath/osv-detector): A scanner that uses the OSV database.
+-   [G-Rath/osv-detector](https://github.com/G-Rath/osv-detector): A scanner
+    that uses the OSV database.
+-   [Rust client library](https://github.com/gcmurphy/osv)
```

Comment:
Also https://github.com/DependencyTrack/dependency-track

Reply:
Thanks, I wonder how that one didn't turn up in my search?